app/error-handler.js

```diff
@@ -2,6 +2,7 @@
 // native support in the browsers, so that our error handler
 // can catch script-loading errors.
 
+/* eslint-disable no-var */
```

Comment:
Even though this file cannot get the full ES6 treatment, I think `let` and `const` are fine. We only had problem with really old Safari in that regard.